src/services/Webhooks.php

```diff
@@ -61,6 +113,13 @@ public function processWebhook(Gateway $gateway): Response
             $mutex->release($transactionLockName);
         }
 
+        // Fire a 'afterProcessWebhook' event
+        if ($this->hasEventHandlers(self::EVENT_AFTER_PROCESS_WEBHOOK)) {
+            $this->trigger(self::EVENT_AFTER_PROCESS_WEBHOOK, new WebhookEvent([
```

Comment:
@pdaleramirez @lukeholder should we include the gateway as well in this event? It could give the developers that extra bit of context.

Thoughts?